lib/tls.js

```diff
@@ -1327,6 +1330,21 @@ function normalizeConnectArgs(listArgs) {
   return (cb) ? [options, cb] : [options];
 }
 
+// Returns true if the --enable-legacy-cipher-list command line
+// switch, or the NODE_LEGACY_CIPHER_LIST environment variable
+// are set to v0.10.38 and the DEFAULT_CIPHERS equal the v0.10.38
+// list.
+function usingV1038Ciphers() {
+  var argv = process.execArgv;
+  if ((argv.indexOf('--enable-legacy-cipher-list=v0.10.38') > -1 ||
+       process.env['NODE_LEGACY_CIPHER_LIST'] === 'v0.10.38') &&
+      DEFAULT_CIPHERS === _crypto.getLegacyCiphers('v0.10.38')) {
+        return true;
+  }
+  return false;
+}
+exports.usingV1038Ciphers = usingV1038Ciphers;
```

Comment:
@misterdjules ... Yes, my original intent was for this to be internal only but given the need to expose this is crypto.js also, this was the quick fix. Perhaps what we need is to push the usingV1038Ciphers into the `process.binding('crypto')` API instead.